.github/workflows/perf.yml

```
@@ -27,14 +27,15 @@ concurrency:
   CARGO_TERM_COLOR: always
   CARGO_INCREMENTAL: "0"
+  TAK_RUNNER: bamboo-v1-ubuntu24.04-x64-rust1.97.1
 
 jobs:
   measure:
     name: Measure
     # Pinned to one runner class on purpose. Absolute instruction counts shift
     # between machine types by more than a real regression does, so a series
     # that wanders between runners is unreadable.
-    runs-on: ubuntu-latest
+    runs-on: bamboo-perf
```

actionlint (1.7.12)

[error] label "bamboo-perf" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows", "macos-14", "buildjet-32vcpu-ubuntu-2204-arm", "buildjet-16vcpu-ubuntu-2204-arm", "buildjet-8vcpu-ubuntu-2204-arm", "buildjet-4vcpu-ubuntu-2204-arm", "buildjet-2vcpu-ubuntu-2204-arm", "buildjet-32vcpu-ubuntu-2204", "buildjet-16vcpu-ubuntu-2204", "buildjet-8vcpu-ubuntu-2204", "buildjet-4vcpu-ubuntu-2204", "buildjet-2vcpu-ubuntu-2204". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)